src/tests/ftest/launch.py

```diff
@@ -35,6 +36,21 @@
 
 TEST_LOG_FILE_YAML = "./data/daos_avocado_test.yaml"
 BASE_LOG_FILE_YAML = "./data/daos_server_baseline.yaml"
+SERVER_KEYS = (
+    "test_machines",
+    "test_servers",
+    "daos_servers",
+    "servers",
+    "test_machines1",
+    "test_machines2",
+    "test_machines2a",
+    "test_machines3",
+    "test_machines6",
+    )
+CLIENT_KEYS = (
+    "test_clients",
+    "clients",
+    )
```

Comment:
We should standardize.  Hopefully the expanded use of apricot will drive this.

Reply:
Hopefully.

Reply:
I wonder if it's relevant (enough) to this patch to just go through the YAML files and standardise these arrays of names and just eliminate these lists of _KEYS_.

I won't -1 this patch for that, but something for you to consider.  I'll leave the final decision up to you.

Reply:
I went through all the current tests and the list of used keys has shrunk down to four different keys for servers (`test_servers`, `test_machines`, `test_machines1`, and `daos_servers`) and one client key (`test_clients`).  The `test_machines1` key is used once by `pool/full_pool_container_create` and it can be fixed easily.  The `daos_servers` key is used only by `soak/soak` which I know Maureen is replacing).  The `test_machines` key is used by 39 tests and although most are using TestWithServers, there are a few holdouts.  These tests also do not explicitly specify hosts to use as clients - therefore using the launch host to run daos_agent.  I think both of these issues should be resolved in a different PR.